Content.Server/GameObjects/EntitySystems/PowerSolarSystem.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Content.Server.GameObjects.Components.Power;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Maths;
+using System;
+
+namespace Content.Server.GameObjects.EntitySystems
+{
+    /// <summary>
+    ///     Responsible for maintaining the solar-panel sun angle and updating <see cref='SolarPanelComponent'/> coverage.
+    /// </summary>
+    [UsedImplicitly]
+    public class PowerSolarSystem: EntitySystem
+    {
+        /// <summary>
+        /// The current sun angle.
+        /// </summary>
+        public Angle TowardsSun = Angle.South;
+
+        public override void Initialize()
+        {
+            EntityQuery = new TypeEntityQuery(typeof(SolarPanelComponent));
+        }
+
+        public override void Update(float frameTime)
+        {
+            TowardsSun += Angle.FromDegrees(frameTime / 10).Reduced();
```

Comment:
This `.Reduced()` applies *before* the addition, meaning that `TowardsSun` will not be reduced itself and keep growing.

Reply:
Oh, right. Whoops. Will fix.

Reply:
Fixed.